src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * Main Duke chatbot class.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    /** List of Tasks */
+    private ArrayList<Task> list;
+    private Duke(ArrayList<Task> list){
+
+        this.list = list;
+
+    }
+
+    public void greeting(){
```

Comment:
```suggestion
    public void sayGreeting() {
```
I think a verb here is good? Also white space